x/gamm/keeper/pool_service_test.go

```diff
@@ -419,7 +418,7 @@ func (suite *KeeperTestSuite) TestJoinPoolNoSwap() {
 			tokenInMaxs: sdk.Coins{
```

Comment:
Ah this test case was wrong, glad that we have this PR + change. This shouldn't be testing with negative shares, it should have positivie shares and we're expecting error related to insufficient funds, not negative share ratio. Would you be able to change this test case within this PR as well? We can also do it in a separate PR if you would prefer that way!

Reply:
I updated in this pr